src/components/Form.js

```diff
@@ -0,0 +1,42 @@
+import {useState} from 'react';
+
+export default function FormPage() {
+	const [invoice, setInvoice] = useState(false);
+	const [name, setName] = useState('');
+	const [amount, setAmount] = useState('');
+
+	return (
+		<div>
+			{invoice ? (
+				<form>
+					<ul>
+						<li>{name}</li>
+						<li>{amount}</li>
+					</ul>
+				</form>
+			) : (
```

Comment:
```suggestion
			) 
```
you want to display the form independently of an already existing invoice right?
So you want to only include the first part into the conditional rendering :)